python/packages/ai/teams/ai/promptsv2/prompt_template.py

```diff
@@ -0,0 +1,27 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from dataclasses import dataclass
+
+from .prompt_section import PromptSection
+from .prompt_template_config import PromptTemplateConfig
+
+
+@dataclass
+class PromptTemplate:
+    """
+    Prompt template cached by the prompt manager.
+
+    Attributes:
+        name (str): Name of the prompt template.
+
+        prompt (PromptSection): Text of the prompt template.
+
+        config (PromptTemplateConfig): Configuration settings for the prompt template.
+    """
+
+    name: str
+    prompt: PromptSection
+    config: PromptTemplateConfig
```

Comment:
If it's not marked down yet, could we add to a ticket (existing or new) to add in the actions and augmentation parameters?

Reply:
Sure. I haven't finish the PromptManager class, will add TODO tickets under related tickets after I figured out all the implementation.

Reply:
Added comments to https://github.com/microsoft/teams-ai/issues/1068 and https://github.com/microsoft/teams-ai/issues/1069